scalafix-core/shared/src/main/scala/scalafix/internal/config/Class2Hocon.scala

```diff
@@ -1,7 +1,19 @@
 package scalafix.internal.config
 
 import metaconfig.HasFields
-import metaconfig.String2AnyMap
+
+object String2AnyMap {
```

Comment:
:cold_sweat:

Reply:
This is a total hack that was previously in metaconfig and I deprecated it 😅